lib/main.js

```diff
@@ -15,6 +16,7 @@ Object.assign(exports, {
 
     // options.loadReason can be install/enable/startup/upgrade/downgrade
     PlacesProvider.links.init();
+    TabTracker.init();
```

Comment:
Let's leave the pref to another bug. For now, a telemetry parameter passed to Activitystreams that doesn't init TabTracker at all is great.